---
title: "casmini-tool: a comprehensive database for efficient and specific guide RNA design using dCasMINI"
collection: publications
category: manuscripts
permalink: /publication/RWY-2024-06-25-paper
excerpt: "Lopp, S.^, Borrman, T.^, Jha, N., Silvis, M., Swan, R., Yang, X., Alvarez, G., Klappenbach, C., Liu, Y., Hart, D. O., Qi, L.S., Daley, T. P.‡, & __Yeo, R.W.‡__  _Biorxiv_  (2024) <br> **‡_co-corresponding authors_**"
date: 2024-06-25
venue: 'Biorxiv'
#slidesurl: 'http://academicpages.github.io/files/slides1.pdf'
paperurl: 'https://www.biorxiv.org/content/10.1101/2023.09.17.558168v2'
#citation: 'Your Name, You. (2009). &quot;Paper Title Number 1.&quot; <i>Journal 1</i>. 1(1).'
---

The dCasMINI protein is a hypercompact, nuclease-inactivated CRISPR-Cas system engineered for transcriptional modulation and epigenetic editing [Xu et al., 2021]. The small size of dCas-MINI (529 amino acids), less than half the size of comparable Cas9 molecules, makes it ideal for AAV-based therapies which are frequently limited by AAV’s small cargo capacity. Unlike Cas9 or Cas12a, there are no available computational tools for designing dCasMINI guides. To facilitate and accelerate the development of dCasMINI-based applications, we synthesized knowledge regarding dCasMINI guide design and built a website to assist researchers in designing optimal guides for dCasMINI-based experiments for transcriptional inhibition (CRISPRi) and activation (CRISPRa); to ensure that our tool would be useful for therapeutic guide design, in which a guide’s off-target safety profile is of paramount importance, we specifically optimized alignment parameters for high-sensitivity to comprehensively report genome-wide off-targets. To investigate dCasMINI’s full protospacer adjacent motif (PAM) profile, we engineered libraries of PAMs and exhaustively characterized dCasMINI’s ability to activate a locus with different PAMs. We also experimentally investigated the importance of each nucleotide position on the guide RNA’s ability to activate its target, and characterized a 6bp high-fidelity seed region at the 5’ end of the protospacer sequence which we identified to be intolerant to mismatches and deletions, and thus critical for true binding events. Taken together, our tool offers CRISPRi/a guide design for every protein-coding gene in the human genome along with comprehensive off-target prediction, incorporating the most up-to-date information about dCasMINI’s full PAM and protospacer design rules. The tool is freely available to use at www.casmini-tool.com.
	